ygot/types.go

```diff
@@ -76,6 +79,17 @@ type GoEnum interface {
 	// generated code file. The ygen library generates a static map of
 	// enumeration values that this method returns.
 	ΛMap() map[string]map[int64]EnumDefinition
+	// String provides the string representation of the enum, which will be
+	// the YANG name if it's in its defined range.
+	String() string
+}
+
+func EnumString(e GoEnum, val int64, enumTypeName string) string {
```

Comment:
Is this covered by any test? It's probably worth adding a simple test somewhere.